contrib/ay-openqa-worker-leap-3-nvme.xml

```diff
@@ -54,6 +54,7 @@
     </patterns>
     <packages t="list">
       <package>openssh</package>
+      <package>chrony</package>
```

Comment:
So was this tested? Did you check if it is linked and documented somewhere?

Reply:
@kalikiana I would trust that this is a trivial change. We have seen a validation for the .xml.erb profile already. Automatic validation will be covered as part of https://progress.opensuse.org/issues/156169 so out of scope for here.